doc/filters/json_encode.rst

```diff
@@ -15,7 +15,9 @@ Arguments
 ---------
 
 * ``options``: A bitmask of `json_encode options`_ (``{{
-  data|json_encode(constant('JSON_PRETTY_PRINT')) }}``)
+  data|json_encode(constant('JSON_PRETTY_PRINT')) }}``. Combine constants using `Twig's bitwise operators`: ``{{
+  data|json_encode(constant('JSON_PRETTY_PRINT') b-or constant('JSON_HEX_QUOT') }}``)
 
 .. _`json_encode`: https://secure.php.net/json_encode
 .. _`json_encode options`: https://secure.php.net/manual/en/json.constants.php
+.. `Twig's bitwise operators`: https://twig.symfony.com/doc/2.x/templates.html#logic
```

Comment:
as that's a doc to the Twig documentation, it should use an internal reference instead, so that it links to the same version of the doc

Reply:
indeed, done inf2b8e9ac